test/e2e/util/test-helper.js

```diff
@@ -2,28 +2,39 @@
 
 process.env.NODE_ENV = 'test';
 
-let supertest = require('supertest');
-
-const options = {
-    databaseUri: require('./database-config').getDatabaseUri(),
-    databaseSchema: 'test'
-};
-
+const supertest = require('supertest');
 const migrator = require('../../../migrator');
 const { createStores } = require('../../../lib/db');
+const { createDb } = require('../../../lib/db/db-pool');
+const _app = require('../../../app');
 
 // because of migrator bug
 delete process.env.DATABASE_URL;
 
-const db = require('../../../lib/db/db-pool').createDb(options.databaseUri);
+function createApp (databaseSchema = 'test') {
+    const options = {
+        databaseUri: require('./database-config').getDatabaseUri(),
+        databaseSchema,
+        minPool: 0,
+        maxPool: 0,
+    };
+    const db = createDb({ databaseUri: options.databaseUri, minPool: 0, maxPool: 0 });
 
-const createApp = db.raw(`CREATE SCHEMA IF NOT EXISTS ${options.databaseSchema}`)
-    .then(() => migrator(options.databaseUri, options.databaseSchema))
-    .then(() => {
-        const stores = createStores(options);
-        const app = require('../../../app')({stores});
-        return { stores, request: supertest(app) };
-    });
+    return db.raw(`CREATE SCHEMA IF NOT EXISTS ${options.databaseSchema}`)
```

Comment:
if we drop schema, test reruns will be slower, but maybe we then could remove the cleanup stuff?

btw know how to get away logging from the migration runner?